test/unit/web_extension_test.rb

```diff
@@ -3,15 +3,15 @@
 describe 'Cron web' do
   include Rack::Test::Methods
 
-  TOKEN = SecureRandom.base64(32).freeze
+  token = SecureRandom.base64(32).freeze
```

Comment:
Why? I don't like it, I prefer to keep the constant

Reply:
Constants defined in blocks "escape" and become `Object::TOKEN`. Alternatively, we can define these constants under explicit namespace